explorer/src/components/BondBreakdown.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import Table from '@mui/material/Table';
+import { useMediaQuery, useTheme } from '@mui/material';
+import TableBody from '@mui/material/TableBody';
+import { useParams } from 'react-router-dom';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import { MainContext } from 'src/context/main';
+
+export function BondBreakdownTable() {
+        const { id }: any = useParams();
+        const { mixnodeDetailInfo, delegations } = React.useContext(MainContext);
+        const [ bonds, setBonds ] = React.useState({
+            delegations: 0,
+            pledges: 0,
+            bondsTotal: 0,
+            denom: 'PUNK',
+        })
```

Comment:
Code is fine but there seems to be an inconsistency between tab spacing in some files? The spacing on this one is much more than others?

Reply:
I wonder if this was just before i moved to the UK mac. On the Swiss mac setup/config i was getting format on save perfect. On this one i've had to bring my own vscode settings file in to enact that. I worry my config isnt playing nice with Mark's _prettier_ - @mmsinclair perhaps i could borrow you for 10-15 (monday or tuesday) to check i'm not vandalising your hard work on the config.

Reply:
These should be setting it correctly:

https://github.com/nymtech/nym/blob/feature/network-explorer-react/explorer/.editorconfig#L10-L11
https://github.com/nymtech/nym/blob/feature/network-explorer-react/explorer/.eslintrc#L84-L85

I looked through the config and what I put in prettier is too narrow (80 chars) - it should be more like 150:
https://github.com/nymtech/nym/blob/feature/network-explorer-react/explorer/.eslintrc#L84-L85

So the whole `explorer` directory probably needs a run of prettier to fix things up - this should be a separate commit.